modules/ROOT/examples/live-demos/uploadcare/index.html

```diff
@@ -1,86 +1,33 @@
-              https://ucarecdn.com/b384b67a-3c73-4629-be5f-07d6a2f49e01/-/resize/300x/ 300w,
-              https://ucarecdn.com/b384b67a-3c73-4629-be5f-07d6a2f49e01/-/resize/500x/ 500w,
-              https://ucarecdn.com/b384b67a-3c73-4629-be5f-07d6a2f49e01/-/resize/750x/ 750w,
-              https://ucarecdn.com/b384b67a-3c73-4629-be5f-07d6a2f49e01/-/resize/1000x/ 1000w,
-              https://ucarecdn.com/b384b67a-3c73-4629-be5f-07d6a2f49e01/-/resize/1250x/ 1250w,
-              https://ucarecdn.com/b384b67a-3c73-4629-be5f-07d6a2f49e01/-/resize/1500x/ 1500w,
-              https://ucarecdn.com/b384b67a-3c73-4629-be5f-07d6a2f49e01/-/preview/ 2000w">
-    <figcaption>Designed by <a href="https://www.freepik.com/">Freepik</a></figcaption>
-  </figure>
-  <p>
-    Step inside, and you&rsquo;ll find bright, open living spaces that blend seamlessly with the outdoors, creating a harmonious living environment that&rsquo;s both relaxing and inspiring.
-  </p>
-  <p>
-    Owning a villa in Mallorca means more than just having a home&mdash;it means living a life of luxury and adventure. From exploring charming villages to dining at world-class restaurants and sailing the turquoise waters, every day is a new experience.
-  </p>
-  <h2>Make This Dream a Reality</h2>
-  <p>
-    Don&rsquo;t miss the opportunity to own a piece of paradise. <a href="https://www.tiny.cloud/contact/">Contact us today</a> to schedule a private tour of this exceptional villa and explore other exclusive properties on the island.
-  </p>
-</textarea>
-  
+   <h1>Discover Your Dream Home on Mallorca: Where Paradise Meets Investment</h1>
+<p><span style="color: #4e5c73;">Mallorca, the jewel of the Mediterranean, is not just a place of beauty&mdash;it&rsquo;s a smart investment. With property values consistently on the rise, now is the perfect time to secure your piece of this sun-soaked paradise.&nbsp;</span></p>
+<figure class="image">
+   <img style="width: min(2048px, 100%); aspect-ratio: 2048 / 718;" src="https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/preview/" sizes="(min-width: 2048px) 2048px, 100vw" srcset="https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/100x/ 100w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/200x/ 200w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/300x/ 300w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/500x/ 500w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/750x/ 750w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/1000x/ 1000w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/1250x/ 1250w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/1500x/ 1500w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/2000x/ 2000w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/preview/ 2048w" alt="Mallorca">
```

Comment:
- [ ] We talked about changing these to tiny.ucarecdn.com urls to be consistent.

Reply:
The latest commit has the updated URLs. So this is resolved.